Client/Frontend/Home/TopSitesPanel.swift

```diff
@@ -10,6 +10,7 @@ import Storage
 private let log = Logger.browserLogger
 
 private let ThumbnailIdentifier = "Thumbnail"
+private let FrecencyLimit = 20
```

Comment:
I guess we have to worry about split-screen on iPads…

What will this do on an iPad Pro? Is 20 big enough?